course02/homework01/linked_list.h

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+using namespace std;
```

Comment:
Please don't do this, it's a bad practice in general, but especially in a header file it's a very bad practice. The problem is that anybody that will include this header will also have this `using namespace std` meaning that you will pollute everybody's code, and not just yours. This could, and will, also break code.